xenit-alfresco/values.yaml

```diff
@@ -97,7 +97,7 @@ postgresql:
 solr:
   enabled: true
   replicas: 2
-  image: 'hub.xenit.eu/alfresco-enterprise/alfresco-solr6:2.0.2'
+  image: 'private.docker.xenit.eu/alfresco-enterprise/alfresco-solr6-xenit:2.0.6'
```

Comment:
I think the default of this helm chart should be community. 
https://hub.docker.com/layers/xenit/alfresco-solr6/2.0.6/images/sha256-8803a5de6d132612342215ca93b0a788e3a8f8701cc11afe75597c4277098069?context=explore